inspirehep/modules/records/jsonschemas/records/elements/id.json

```diff
@@ -0,0 +1,155 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "anyOf": [
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "INSPIRE ID"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "INSPIRE-\\d+",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "INSPIRE BAI"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "(\\w+\\.)+\\d+",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "ORCID"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "\\d{4}-\\d{4}-\\d{4}-\\d{3}[0-9X]",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "JACOW"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "JACoW-\\d+",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "KAKEN"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "\\d+",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "ARXIV"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "\\w+",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "CERN"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "CCID-\\d+",
```

Comment:
*Peu importe*. I'd vote for CERN, since CCID is how they are internally called at CERN but it's not known outside.

For the limit, in theory nope, because they are simply an autoincrement integer.